circle.yml

```
@@ -1312,7 +1312,7 @@ jobs:
   test-npm-module-on-minimum-node-version:
     <<: *defaults
     docker:
-      - image: cypress/base:10.0.0
+      - image: cypress/base:12.0.0
```

```suggestion
      - image: cypress/base:12.0.0-libgbm
```

The original base-12.0.0 image didn't include libgbm, this should fix the failing CI stage